src/maintenance/update_last_usage.py

```diff
@@ -0,0 +1,42 @@
+import os
+from datetime import datetime as dtime
+
+import delphi.operations.secrets as secrets
+import mysql.connector
+import redis
+
+REDIS_HOST = os.environ.get("REDIS_HOST", "delphi_redis")
+REDIS_PASSWORD = os.environ.get("REDIS_PASSWORD", "1234")
+LAST_USED_KEY_PATTERN = "*LAST_USED*"
+
+
+def redis_get_last_time_used(redis_cli):
+    keys = redis_cli.keys(pattern=LAST_USED_KEY_PATTERN)
+    key_val_pairs = {
+        str(key).split("/")[1]: dtime.strptime(str(redis_cli.get(key)), "%Y-%m-%d").date() for key in keys
+    }
+    return key_val_pairs
+
+
+def main():
+    redis_cli = redis.Redis(host=REDIS_HOST, password=REDIS_PASSWORD, decode_responses=True)
+    key_val_pairs = redis_get_last_time_used(redis_cli, LAST_USED_KEY_PATTERN)
```

Comment:
to match the new function signature for `redis_get_last_time_used()`:
```suggestion
    key_val_pairs = redis_get_last_time_used(redis_cli)
```

Reply:
Oops, missed that. Thx!